orm/model/ormlist/options.go

```diff
@@ -40,9 +40,14 @@ func Cursor(cursor CursorT) Option {
 // The Iterator.PageRequest value on the returned iterator will be non-nil
 // after Iterator.Next() returns false when this option is provided.
 // Care should be taken when using Paginate together with Reverse and/or Cursor
-// and generally this should be avoided.
+// and generally this should be avoided. If pageRequest is nil, this option
+// will be a no-op.
```

Comment:
so there won't be a return limit? Maybe it's worth to add that limits should be used

Reply:
we could also specify a default limit on the table/db level so that it isn't needed in every query. what do you think?

Reply:
is there a gas limit for queries?

Reply:
ah, good point. Queries can kill the node.